test/unit/specs/common/filters/cd-url-formatter.spec.js

```diff
@@ -0,0 +1,10 @@
+import cdUrlFormatter from '@/common/filters/cd-url-formatter';
+
+describe('Url formatting filter', () => {
+  it('should format url to include http:// or https://', () => {
+    expect(cdUrlFormatter('www.example.com')).to.equal('http://www.example.com');
+    expect(cdUrlFormatter('http://www.example.com')).to.equal('http://www.example.com');
+    expect(cdUrlFormatter('https://www.example.com')).to.equal('https://www.example.com');
+    expect(cdUrlFormatter('http:/www.example.com')).to.equal('http://http:/www.example.com');
```

Comment:
I guess it's a slightly pointless test, but I get the idea behind it. It has to be a full valid URL or it gets prefixed. Really this tests the same as the first test, but I guess this additionally checks that we're not being really naïve and checking that the string starts with 'http'